common/process_test.cc

```diff
@@ -313,11 +313,10 @@ TEST_F(ProcessTest, LogOutputLevelDetection) {
 }
 
 TEST_F(ProcessTest, Terminate) {
+  // Use ping to simulate a sleep instead of timeout since timeout fails with
+  // "Input redirection is not supported".
   ProcessStartInfo start_info;
-  start_info.command = "timeout /T 30";
-  // Prevents the process from shutting down immediately when run in background.
-  // https://www.ibm.com/support/pages/timeout-command-run-batch-job-exits-immediately-and-returns-error-input-redirection-not-supported-exiting-process-immediately
-  start_info.redirect_stdin = true;
+  start_info.command = "ping -n 30 127.0.0.1";
```

Comment:
On Linux, the corresponding flag is `-c`. May I suggest to move the cmd into a helper function and make it work for both Linux and Windows?

Reply:
The class itself and almost every test is Windows specific right now (calls to Windows API, cmd /C etc.). I'm hesitant to make this one issue work on Linux as it could give the false impression that the class supports Linux. Instead, it might make more sense to generalize this test when support for Linux processes is added.